pydoctor/test/epydoc/test_restructuredtext.py

```diff
@@ -26,3 +29,80 @@ def test_rst_anon_link_email() -> None:
     assert html.startswith('<a ')
     assert ' href="mailto:postmaster@example.net"' in html
     assert html.endswith('>mailto:postmaster@example.net</a>')
+
+# THIS IS HELPER METHODS
+from pydoctor.epydoc.markup import flatten, restructuredtext
+def to_html(docstring) -> str:
+    """
+    Utility method to convert a docstring to html with pydoctor. 
+    """
+    err = []
+    p = restructuredtext.parse_docstring(docstring, err)
+    if [ e.__dict__ for e in err ]!=[]: 
```

Comment:
```suggestion
    if err:
```
This is testing whether there were any errors, right?

Reply:
Yep, I'll change that